src/System.Management.Automation/security/Authenticode.cs

```diff
@@ -292,7 +292,7 @@ internal static Signature GetSignature(string fileName, string fileContent)
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods")]
         private static Signature GetSignatureFromCatalog(string filename)
         {
-            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.Value)
+            if (!Signature.CatalogApiAvailable.GetValueOrDefault())
```

Comment:
Checking HasValue is important here
```suggestion
            if (Signature.CatalogApiAvailable.HasValue && !Signature.CatalogApiAvailable.GetValueOrDefault())
```